portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +33,92 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> quotes;
```

Comment:
We're no longer using the ImmutableList?

If the issue is that the previous branch isn't reflecting on this one, you can merge it into this one and you should be good.